jekyll-assets/css/style.css

```diff
@@ -361,6 +361,7 @@ h6 .anchor::before {
   font-style: italic;
   margin-top: -12px;
   margin-bottom: 20px;
+  margin-left: 5px;
```

Comment:
Errr, that's "pushed 5 pixels to the right" :stuck_out_tongue_winking_eye:

Reply:
Left, right, all the same. Yes. 5 pixels to the right. 🤦‍♂️